### ISSUE TITLE
Ansible for use deploying Circles?

### ISSUE THREAD

**Issue by jlariviere1** [circles-project/matrix-docker-ansible-deploy#9]
Can I use the ansible-deploy to set up a matrix system for Circles?  I thought I heard there were some extra steps to configure my own server to support my own Circles (now that Futo Circles server is being shutdown), but don't see documentation for that.  
Any help for that would be great! 
Thanks!

**Comment by Michael-Hollister** [comment]
Yes you can. And you are correct, there are some additional steps for setting up the server for it to be compatible with the Circles apps (At least for Android users, the iOS Circles app is more flexible and does not require some of the additional configuration).

I was in the process of making a wiki page documenting the setup process and its almost done, but I am currently trying to find a workaround on getting registered user accounts working on the Circles Android app. Once that issue is resolved I will be posting the wiki page.

**Comment by Michael-Hollister** [comment]
Got around to finishing the wiki page. You can find the setup guide here: https://github.com/circles-project/matrix-docker-ansible-deploy/wiki/Circles-Server-Setup
